connector_jira_servicedesk/i18n/connector_jira_servicedesk.pot

```diff
@@ -0,0 +1,161 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* connector_jira_servicedesk
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```

Comment:
Maybe
```suggestion
"Project-Id-Version: Odoo Server 13.0\n"
```

Reply:
Regenerate the whole translation just to be sure ^^

Reply:
not needed. POT will be re-generated